tsconfig.json

```diff
@@ -152,14 +195,36 @@
       "@spartacus/checkout/scheduled-replenishment/root": [
         "feature-libs/checkout/scheduled-replenishment/root/public_api"
       ],
-      "@spartacus/order/assets": ["feature-libs/order/assets/public_api"],
+      "@spartacus/customer-ticketing/assets": [
+        "feature-libs/customer-ticketing/assets/public_api"
+      ],
+      "@spartacus/customer-ticketing/core": [
+        "feature-libs/customer-ticketing/core/public_api"
+      ],
+      "@spartacus/customer-ticketing": [
```

Comment:
missing "@spartacus/customer-ticketing/occ"

as mentioned in a previous comment to create